openhands-tools/openhands/tools/terminal/definition.py

```diff
@@ -196,6 +196,7 @@ def visualize(self) -> Text:
 * Persistent session: Commands execute in a persistent shell session where environment variables, virtual environments, and working directory persist between commands.
 * Soft timeout: Commands have a soft timeout of 10 seconds, once that's reached, you have the option to continue or interrupt the command (see section below for details)
 * Shell options: Do NOT use `set -e`, `set -eu`, or `set -euo pipefail` in shell scripts or commands in this environment. The runtime may not support them and can cause unusable shell sessions. If you want to run multi-line bash commands, write the commands to a file and then run it, instead.
+* Shell configuration: By default, the tool auto-detects bash from PATH. You can customize the shell binary by passing the `shell_path` parameter when creating the TerminalTool
```

Comment:
I don't think we need to add this here -- the agent didn't and shouldn't have a way to set it 🤔